### ISSUE TITLE
Create first technical setup

### ISSUE THREAD

**Issue by dot-net-zuid** [dot-net-zuid/publicwebsite#2]
There needs to be an initial technical setup that determines the shape of the solution.

**Comment by Creations2Be** [comment]
Using Active Directory for storage of user-info (attendee, organizer or admin) would be my first choice. But since this is a public repo - and one would have to be able to code against this AD - I'd suggest separate AD's for DEV and PROD.

**Comment by dot-net-zuid** [comment]
Agreed, that would be ideal. That would be the case for more assets, like a database, storage or whatever else is needed

**Comment by rajbos** [comment]
Whats the advantage of using AAD instead of just letting users register with their social accounts? Google/Facebook/Github/etc? 

Would we only use it for storage and tack to the AAD Api? 

AAD costs for resetting the passwords and 2FA are not cheap and maybe a little though to set up for someone on their first try.

**Comment by dot-net-zuid** [comment]
Even if you have people authenticate with external accounts like Facebook, Google, etc, you'd need some place to organize their access to the site. We have this at the moment in the database, which is a security risk and not meant for that purpose.

If we use AAD, we can have first-party accounts (actual custom accounts people create for dotnetzuid.nl) and potentially also third-party accounts, although the later is of less priority for now. 

Don't worry about the costs for now, we are sponsored by Microsoft for the Azure costs.

**Comment by dot-net-zuid** [comment]
By the way, if anyone wants to start and create the initial technical setup for the solution, please indicate that you are working on this so that we can assign you to this issue. That way, we'll start out with one setup.

**Comment by jphellemons** [comment]
I prefer social accounts like @rajbos says. There are good .net Core docs about it and the default template has some identity stuff pre configured. It was discussed in one of the last .net community standups too.

**Comment by Creations2Be** [comment]
I could have a basic ASP.NET Core Mvc 2.0 website with a basic design (and some content from the current website) up-and-running in a few days. This setup would not support login yet but could be deployed to Azure (or run locally) without any need for a database.

**Comment by bmaluijb** [comment]
That would be excellent! I’ll make you a contributor, so that you have commit rights.

**Comment by bmaluijb** [comment]
Hi @Creations2Be, are you working on this? Can I assign you to it?

**Comment by Creations2Be** [comment]
Yes @bmaluijb, I am... and yes you can.

**Comment by bmaluijb** [comment]
@Creations2Be Any progress on this one?

**Comment by Creations2Be** [comment]
Hi @bmaluijb, last few weeks have been pretty hectic but I'll commit my work before the end of the week.

**Comment by Creations2Be** [comment]
I've had some stability-issues during testing after the latest VS2017 update resulting in some delay of the commit.

**Comment by bmaluijb** [comment]
Alright, looking forward to your commit!
